content/en/about/faq/traffic-management/statefulsets.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Can Istio be used with StatefulSets and headless Services?
```

Comment:
I think the capitalization of headless should match services. I'm in favor of uppercasing the `h`.

Reply:
From k8s docs: "In this case, you can create what are termed "headless" Services". "headless" is not a proper noun while Services and StatefulSets are referring to the object names?

Reply:
Interesting as I found k8s docs like, https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/, that have `StatefulSets currently require a Headless Service to be responsible...`

I'll take your comment as a guide.